src/NotificationsApp.vue

```diff
@@ -1,16 +1,12 @@
 <template>
-	<div v-if="!shutdown" class="notifications">
-		<div ref="button"
-			class="notifications-button menutoggle"
-			:class="{ hasNotifications: notifications.length }"
-			tabindex="0"
-			role="button"
-			:aria-label="t('notifications', 'Notifications')"
-			aria-haspopup="true"
-			aria-controls="notification-container"
-			aria-expanded="false"
-			@click="requestWebNotificationPermissions"
-			@keydown.enter="requestWebNotificationPermissions">
+	<HeaderMenu v-if="!shutdown"
+		id="notifications"
+		class="Notifications"
```

Comment:
The dot is not red anymore if there is a new notification.
```suggestion
		class="notifications-button"
```

But then positions are off, so maybe most of the rules directly in `notifications-button` should be removed?